multiprocessing_task.py

```diff
@@ -0,0 +1,83 @@
+import multiprocessing
+import time
+
+N = 300000000
+
+
+def simple_iteration():
+    start = time.time()

```

Comment:
Could you please use @timeit decorator? As it was in the task :)

Reply:
Done